internal/provisioner/cluster_installation_provisioner.go

```diff
@@ -1097,3 +1136,12 @@ func makeClusterInstallationName(clusterInstallation *model.ClusterInstallation)
 	// of 60 characters.
 	return fmt.Sprintf("mm-%s", clusterInstallation.Namespace[0:4])
 }
+
+func contains(slice []string, value string) bool {
+	for _, v := range slice {
+		if v == value {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Will this existing util function work instead? https://github.com/mattermost/mattermost-cloud/blob/aa6c3bcd9f0a4b92d12762bca69880b2d7c8a4de/internal/common/util.go#L8